src/main/java/teammates/ui/template/InstructorFeedbackResultsResponsePanel.java

```diff
@@ -0,0 +1,103 @@
+package teammates.ui.template;
+
+import java.util.List;
+
+import teammates.common.datatransfer.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.FeedbackResponseAttributes;
+
+public class InstructorFeedbackResultsResponsePanel {
+    private FeedbackQuestionAttributes question;
+    private FeedbackResponseAttributes response;
+    
+    private String questionText;
+    private String additionalInfoText;
+    
+    private ElementTag rowAttributes;
+    
+    private String displayableResponse;
+    
+    private List<FeedbackResponseComment> comments;
+    private FeedbackResponseComment frcForAdding;
+    private boolean isAllowedToAddComment;
+    
+    // The indexes are used for the parameters of js functions for handling response comments 
+    private int recipientIndex;
+    private int giverIndex;
+    private int qnIndex; // TODO  investigate using question number instead of tracking an index
+    
+    public InstructorFeedbackResultsResponsePanel(FeedbackQuestionAttributes question, FeedbackResponseAttributes response,
+                                                  String questionText, String additionalInfoText, 
+                                                  ElementTag rowAttributes, 
+                                                  String displayableResponse, 
+                                                  List<FeedbackResponseComment> comments, boolean isAllowedToAddComment) {
+        this.question = question;
+        this.response = response;
+        this.questionText = questionText;
+        this.additionalInfoText = additionalInfoText;
+        this.rowAttributes = rowAttributes;
+        this.displayableResponse = displayableResponse;
+        this.comments = comments;
+        this.isAllowedToAddComment = isAllowedToAddComment;
+    }
+    
+    public void setFeedbackResponseCommentsIndexes(int recipientIndex, int giverIndex, int qnIndex) {
+        this.recipientIndex = recipientIndex;
+        this.giverIndex = giverIndex;
+        this.qnIndex = qnIndex;
+    }
```

Comment:
I think `setCommentsIndexes()` or `setIndexes` should be enough for this name.
There's enough context since it's already in the `InstructorFeedbackResultsResponsePanel` class and all.